doc/OnlineDocs/working_models.rst

```diff
@@ -399,16 +455,21 @@ Access to paramaters is completely analgous to access to variables. For
 example, here is a code snippet to print the name and value of every
 Parameter:
 
-.. literalinclude:: script_spy_files/spy4scripts_Print_parameter_name&value.spy
-   :language: python
+   >>> for parmobject in instance.component_objects(pyo.Param, active=True):
+   ...     nametoprint = str(str(parmobject.name))
+   ...     print ("Parameter ", nametoprint)  # doctest: +SKIP
+   ...     for index in parmobject:
+   ...         vtoprint = pyo.value(parmobject[index])
+   ...         print ("   ",index, vtoprint)  # doctest: +SKIP
 
 .. note::
 
-   The value of a ``Param`` can be returned as None+ if no data was
+   The value of a ``Param`` can be returned as ``None`` if no data was
    specified for it. This will be true even if a default value was
-   given. To inspect the default value of a ``Param``, replace
-   ``.value`` with ``.default()`` but note that the default might be a
-   function.
+   given. To inspect the default value of a ``Param``, use
```

Comment:
I'm not sure that this note is still true. I did a quick test with mutable and immutable parameters and couldn't find a case where `None` was returned when a default value was specified.